demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -661,7 +679,9 @@ def test_run_all_validations_on_file_failed(self, mocker, file_path):
         validate_manager = ValidateManager(file_path=file_path, skip_conf_json=True)
         assert not validate_manager.run_validation_on_specific_files()
 
-    def test_run_all_validations_on_file_with_modified_id(self, mocker, integration):
+    def test_run_all_validations_on_file_with_modified_id(
+        self, set_git_test_env, mocker, integration
```

Comment:
please add type hints to all modified function signatures

Reply:
Does it not support type hints? 🤔

Reply:
For some reason when I'm trying to give it the hint "pytest.fixture" it fails